src/components/shared/Drawer.jsx

```diff
@@ -0,0 +1,84 @@
+import Link from 'next/link';
+import { FaSearch } from 'react-icons/fa';
+import './Header.css';
+
+const Drawer = ({ setOpenDrawer, user, openDrawer }) => {
+  return (
+    <div
+      className={`slide ${openDrawer ? 'open' : ''} min-h-[100vh] w-[70vw]  overflow-hidden bg-california-50 p-4 text-sushi-700 lg:w-[30vw]`}
+    >
+      <div className='flex justify-end'>
+        <button onClick={() => setOpenDrawer(false)}>X</button>
```

Comment:
I can see you use react-icons for {FaSearch}, so we can use cross-icon from react-icons without X characters

Reply:
I have changed it to an icon. Thank you.